tests/suite/scale_test.go

```diff
@@ -797,8 +799,6 @@ var _ = Describe("Zero downtime scale test", Ordered, Label("nfr", "zero-downtim
 	})
 
 	AfterAll(func() {
-		_, err := fmt.Fprintln(outFile)
```

Comment:
What does removing this line mean for us?

Reply:
This was adding an unnecessary newline at the end of the file.

Reply:
Wait, I did this wrong actually. Standby